docs/reference/data/understanding-database.md

```diff
@@ -0,0 +1,54 @@
+# Understanding Database
+
+By default, n8n uses SQLite as the database. The credentials, workflows, executions, and webhook are decoupled and store in different tables.
+
+## Credentials Entity
+
+The credentials get stored in the `credentials_entity` table with the following fields.
+
+| Column Name | Data Type | Description |
+|-------------|-----------|-------|
+| id | INTEGER | auto-incremented and unique for each row |
+| name | VARCHAR | stores the name of the credentials given by the user |
+| data | Text | stores the encrypted data |
+| type | VARCHAR | refers to the credential type used by the node |
+| nodeAccess | TEXT | stores the information of the nodes that have access to these credentials |
+| createdAt | DATETIME | stores the date and time when a credential gets created |
+| updatedAt | DATETIME | stores the date and time when a credential gets updated |
+
```

Comment:
Can you add what an example row would look like here and explain that a bit? (for all the tables)